### PR TITLE
Tests: Fix regression tests

### PR DESCRIPTION
**Pull request by mike10ca**
## How this PR fixes it

- Add minor fixes to regression tests

## How to test it

- Run Cypress tests and observe outcome

### CODE REVIEW

File: cypress/e2e/regression/remove_owner.cy.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ describe('Remove Owners tests', () => {
     owner.verifyRemoveBtnIsEnabled().should('have.length', 2)
   })
 
-  it('Verify Tooltip displays correct message for Non-Owner', () => {
+  //TODO: Clarify correct behaviour for non-owner button visibility
+  it.skip('Verify remove owner button is disabled for Non-Owner', () => {
```
Comment (liliya-soroka) [inline cypress/e2e/regression/remove_owner.cy.js:29]: what an issue do we have currently with the case? Not possible to disconnect signer?

File: cypress/e2e/regression/swaps_history.cy.js [hunk 2]
```diff
@@ -36,16 +36,14 @@ describe('[SMOKE] Swaps history tests', () => {
...
-    create_tx.clickOnTransactionItemByName('11:14 AM')
+    create_tx.clickOnTransactionItemByName('14')
     create_tx.verifyExpandedDetails([
       swapsHistory.sellOrder,
       swapsHistory.sell,
       swapsHistory.oneCOW,
       swapsHistory.forAtLeast,
       swapsHistory.dai,
       swapsHistory.filled,
-      swapsHistory.actionApprove,
```
Comment (liliya-soroka) [inline cypress/e2e/regression/swaps_history.cy.js:47]: why do we remove the check for actions?

File: cypress/e2e/regression/remove_owner.cy.js (hunk 1 above)
Comment (mike10ca) [reply cypress/e2e/regression/remove_owner.cy.js]: This is now updated as discussed.

File: cypress/e2e/regression/swaps_history.cy.js (hunk 2 above)
Comment (mike10ca) [reply cypress/e2e/regression/swaps_history.cy.js:47]: This is now updated as discussed.

**Review by liliya-soroka — APPROVED** [review]

**Pull request merged**